### Some Interesting Resources on Single Cell ☣️

#### [01. Nature Review Single Cell Omics Series](https://www.nature.com/collections/sxnwgntqsk)
Recent technological advances are providing unprecedented opportunities to analyse the complexities of biological systems at the single-cell level. Various crucial biological phenomena are either invisible or only partially characterized when interrogated using standard analyses that average data across a bulk population of cells. However, high-throughput analyses of the genomes, transcriptomes and proteomes of single cells are providing novel and important insights into diverse processes such as development, gene-expression dynamics, tissue heterogeneity and disease pathogenesis.

#### [02. SEURAT - R toolkit for single cell genomics](https://satijalab.org/seurat/)
Seurat is an R package designed for QC, analysis, and exploration of single-cell RNA-seq data. Seurat aims to enable users to identify and interpret sources of heterogeneity from single-cell transcriptomic measurements, and to integrate diverse types of single-cell data.

#### [03. Scanpy – Single-Cell Analysis in Python](https://scanpy.readthedocs.io/en/stable/)
Scanpy is a scalable toolkit for analyzing single-cell gene expression data built jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference and differential expression testing. The Python-based implementation efficiently deals with datasets of more than one million cells.


#### [04. SCVI - Probabilistic models for single-cell omics data](https://scvi-tools.org/)
scvi-tools accelerates data analysis and model development, powered by PyTorch and AnnData.


#### [05. scDeepSort -Cell-type Annotation for Single-cell Transcriptomics using Deep Learning with a Weighted Graph Neural Network](https://github.com/ZJUFanLab/scDeepSort)
A pre-trained cell-type annotation method, namely scDeepSort, using a state-of-the-art deep learning algorithm, i.e. a modified graph neural network (GNN) model. In brief, scDeepSort was constructed based on our weighted GNN framework and was then learned in two embedded high-quality scRNA-seq atlases containing 764,741 cells across 88 tissues of human and mouse, which are the most comprehensive multiple-organs scRNA-seq data resources to date
